src/app/containers/ATIAnalytics/README.md

```diff
@@ -2,56 +2,202 @@
+
+If not done already, Request and Service Contexts will need to be imported to create the tracking URL. To do so, import `buildATIClickParams` and use it within the container to get props required for sending the event beacon.
+
+```jsx
+import React, { useContext } from 'react';
+import { buildATIClickParams } from '../ATIAnalytics/params';
+import { EventContext } from '#contexts/EventContext';
+import { RequestContext } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const componentContainer = (navigationItems) => {
+	const { useClickTracker } = useContext(EventContext);
+	const eventTrackingProps = buildATIClickParams(
+		{},
+		useContext(RequestContext),
+		useContext(ServiceContext)
+	);
 
-```js
-useEffect(() => {
-    sendViewBeacon({
-      ...params,
-      element: document.querySelector('header'),
-      component: 'cookie-banner',
-    });
-}, []);
+
+	useClickTracker('[data-navigation]', event => {
+		// `sendEventBeacon` will go here
+	});
+
+	return (
+		<div className="navigation">
+			<ul className="navigation_list">
+				{navigationItems.map(item => (
+					<li
+						key={item.title}
+						data-navigation
+					>
+						{item.title}
+					</li>
+				))}
+			</ul>
+		</div>
+	);
+}
 ```
 
-## Beacons
-
-The `sendViewBeacon` used in the above example, is meant for impression events, such as the one sent when an element is displayed.
-
-The `sendEventBeacon` is another type of beacon, meant for events such as clicks, scrolls, etc. It sends an impression beacon, right before the event beacon.
-
-```js
-const { useClickTracker } = useContext(EventContext);
-
-useClickTracker('[data-consent-banner] [data-consent-accept]', e => {
-    sendEventBeacon({
-        ...params,
-        element: e.target,
-        component: 'cookie-banner',
-        type: 'click',
-        label: 'cookie-accept',
-    });
-});
-
-useClickTracker('[data-consent-banner] [data-consent-reject]', e => {
-    sendEventBeacon({
-        ...params,
-        element: e.target,
-        component: 'cookie-banner',
-        type: 'click',
-        label: 'cookie-reject',
-    });
-});
+To get all the information about the component and the event, `getComponentInfo` will have to be used. This function takes in 3 parameters - the result, name of the component, and component data.
+Component data should be an object containing information obtained from data attributes.
+The result is a string of the result of having clicked, eg the url of the link or the name of the video being played.
+
+The component data object should contain a value for `actionLabel`, `child`, and optionally `source`. These values will be used as part of the tracking URL.
+- source: The source of the event at a high level. Examples include an urn, or 'responsive_web'
```

Comment:
Can we put the `source` in the example below?

Reply:
This is set by default to `responsive_web~news-simorgh`. For setting to an urn, I think that is more for clicking on a video or maybe something in an article, so in the example I think we'd want to leave it as a default.

I've added a comment both to this line and in the example code to try to make that clear.